server/services/interventionsService.ts

```diff
@@ -315,4 +344,54 @@ export default class InterventionsService {
       headers: { Accept: 'application/json' },
     })) as PCCRegion[]
   }
+
+  async createDraftActionPlan(token: string, referralId: string): Promise<DraftActionPlan> {
+    const restClient = this.createRestClient(token)
+
+    try {
+      return (await restClient.post({
+        path: '/draft-action-plan',
+        headers: { Accept: 'application/json' },
+        data: { referralId },
+      })) as DraftActionPlan
+    } catch (e) {
+      throw this.createServiceError(e)
+    }
+  }
+
+  async getDraftActionPlan(token: string, actionPlanId: string): Promise<DraftActionPlan> {
```

Comment:
There is also a getActionPlan which returns the action plan regardless of whether created or submitted - a general retrieval

get /action-plan/{id}

Reply:
ah ok - does this `draft-action-plan/${id}` exist yet, or is that extra work for you? I think having a separate endpoint is a bit easier to work with, as we don't need to track as much state on the `draftActionPlan` object (and that's how we've done it for `draftReferral`s/`sentReferral`s. Am trying to think of where we might want to use this `/action-plan` endpoint.

Reply:
both GET /action-plan/{id} and /draft-action-plan/{id} exist

the latter is constrained to only draft plans